src/Storage/Storage.Management/help/New-AzDataLakeGen2Item.md

```diff
@@ -0,0 +1,348 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/az.storage/new-azdatalakegen2item
+schema: 2.0.0
+---
+
+# New-AzDataLakeGen2Item
+
+## SYNOPSIS
+Create a file or folder in a container.
+
+## SYNTAX
+
+### File (Default)
+```
+New-AzDataLakeGen2Item [-Container] <String> [-Path] <String> -Source <String> [-Umask <String>]
+ [-Permission <String>] [-Property <Hashtable>] [-Metadata <Hashtable>] [-Force] [-AsJob]
+ [-Context <IStorageContext>] [-ServerTimeoutPerRequest <Int32>] [-ClientTimeoutPerRequest <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [-ConcurrentTaskCount <Int32>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### Folder
+```
+New-AzDataLakeGen2Item [-Container] <String> [-Path] <String> [-Folder] [-Umask <String>]
+ [-Permission <String>] [-Property <Hashtable>] [-Metadata <Hashtable>] [-Force] [-AsJob]
+ [-Context <IStorageContext>] [-ServerTimeoutPerRequest <Int32>] [-ClientTimeoutPerRequest <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [-ConcurrentTaskCount <Int32>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzDataLakeGen2Item** cmdlet creates a file or folder in a container in an Azure storage account.
+This cmdlet only works if Hierarchical Namespace is enabled for the Storage account.
+
+## EXAMPLES
+
+### Example 1: Create a folder with specified Permission, Umask, properties, and metadata
+```
+PS C:\>New-AzDataLakeGen2Item -Container "testcontainer" -Path "dir1/dir2/" -Folder -Permission rwxrwxrwx -Umask ---rw---- -Property @{"CacheControl" = "READ"; "ContentDisposition" = "True"} -Metadata  @{"tag1" = "value1"; "tag2" = "value2" }
+
+   Container Uri: https://storageaccountname.blob.core.windows.net/container1
```

Comment:
Should we have any context about how the account 'storageaccountname' comes to this cmdlet?

Reply:
This is aligned with other dataplane cmdlets.